.github/workflows/deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy to Kubernetes
+
+on:
```

Comment:
`on:
  push:
    branches: [ "main" ]
  pull_request:
    branches: [ "main" ]`

I would make it for PR onto main so if we merge something we know it doesnt break

Reply:
we need a staging for that
different url for staging so that it doesn't take down the site